src/main/java/chess/dao/BoardDao.java

```diff
@@ -0,0 +1,70 @@
+package chess.dao;
+
+import chess.domain.GameState;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.Reader;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import org.apache.ibatis.jdbc.ScriptRunner;
+
+public class BoardDao {
+
+    private static final String INIT_BOARD_FILE_PATH = "/src/main/java/chess/dao/init-board.sql";
+
+    public int initBoard() {
+        final Connection connection = CommonDao.getConnection();
+        final ScriptRunner scriptRunner = new ScriptRunner(connection);
+        final String filePath = System.getProperty("user.dir") + INIT_BOARD_FILE_PATH;
+        final String sql = "SELECT id FROM board ORDER BY id DESC LIMIT 1";
+        try (Reader reader = new BufferedReader(new FileReader(filePath))) {
+            final StatementMaker<PreparedStatement> statementMaker = (statement -> {
+                scriptRunner.runScript(reader);
+            });
+            return CommonDao.findId(sql, statementMaker, "id");
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return CommonDao.FAILED;
+    }
```

Comment:
```suggestion
        } catch (IOException e) {
            e.printStackTrace();
            throw e;
        }
    }
```

예외 상황이 발생했을 떄의 리턴값 때에 대한 처리가 없어서 지금은 예외 상황이 발생했을 때의 반환 값이 다른 로직에도 영향을 주고 있어요.
이런 경우에는 예외를 그대로 또는 다른 예외로 바꿔서 던지는 걸 추천드립니다.

Reply:
찾아보니까 스파크가 exception을 매핑할 수 있는 기능을 제공하고 있네요. 이를 활용해 수정하겠습니다. 또 한, 에러를 외부에서 처리하기 위해 checkedException을 사용하면 해당 흐름에서 사용되는 모든 메서드가 throws를 해야 하므로, 대신에 uncheckedException을 사용하겠습니다.